custom_components/tuya_local/devices/DH-CSK03W_dehumidifier.yaml

```diff
@@ -0,0 +1,95 @@
+name: "DH-CSK03W Dehumidifier"
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 2
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 40
+        max: 70
+      mapping:
+        - step: 5
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "mode_strong"
+          value: "Strong"
+        - dps_val: "mode_sleep"
+          value: "Sleep"
+    - id: 6
+      type: integer
+      name: current_humidity
+secondary_entities:
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 101
+        type: string
+        name: option
+        optional: true
+        mapping:
+          - dps_val: "cancel"
+            value: "Off"
+          - dps_val: "2h"
+            value: "2 hours"
+          - dps_val: "4h"
+            value: "4 hours"
+          - dps_val: "8h"
+            value: "8 hours"
+          - dps_val: "12h"
+            value: "12 hours"
+          - dps_val: "18h"
+            value: "18 hours"
+          - dps_val: "24h"
+            value: "24 hours"
+          - dps_val: "36h"
+            value: "36 hours"
+          - dps_val: "48h"
+            value: "48 hours"
+  - entity: select
+    name: "Light mode"
+    category: config
+    dps:
+      - id: 31
+        type: string
+        name: option
+        optional: true
+        mapping:
+          - dps_val: "ligth_off"
```

Comment:
This misspelling is by tuya?

Reply:
It's an error on their side - if I set it to `light_off` it blanks out when I use the Smart Life app to switch Light modes. Otherwise works properly with the incorrect spelling.